SourceKitStressTester/Tests/StressTesterToolTests/StressTesterToolTests.swift

```diff
@@ -111,6 +111,7 @@ class StressTesterToolTests: XCTestCase {
                                       rewriteMode: .none,
                                       conformingMethodsTypeList: [],
                                       page: Page(),
+                                      offsetFilter: nil,
```

Comment:
The test above isn't the greatest, but probably worth adding `--offset-filter` there.

Reply:
Good idea 👍